psydac/api/ast/utilities.py

```diff
@@ -182,6 +225,46 @@ def compute_atoms_expr_field(atomic_exprs, indices_quad,
                             idxs, basis,
                             test_function, mapping):
 
+    """
+    This function computes atomic expressions needed
+    to evaluate  EvaluteField/VectorField final expression
+
+    Parameters
+    ----------
+    atomic_exprs : <list>
+        list of atoms
+
+    indices_quad : <list>
+        list of quadrature indices used in the quadrature loops
+
+    idxs : <list>
+        list of basis functions indices used in the for loops of the basis functions
+
+
+    basis : <list>
+        list of basis functions in each dimesion
+
+    test_function : <Symbol>
+        test_function Symbol
+
+    mapping : <Mapping>
+        Mapping object
+
+    Returns
+    -------
+    inits : <list>
+       list of assignments of the atomic expression evaluated in the quadature points
+
+    updates : <list>
+        list of augemented assignments which are updated in each loop iteration
+
+    map_stmts : <list>
+        list of assigments of atomic expression in case of mapping
+
+    atomic_exprs: <list>
+        list of new atomic expression in case of mapping that were introduced in case of a mapping
```

Comment:
I cannot understand this explanation: maybe you are repeating the words `in case of a mapping`?